easy-ui-icons/vite.config.ts

```diff
@@ -87,15 +87,14 @@ async function getSvgFilenameFromJson(filename: string) {
 function svgTypeFileTemplate(svgName: string) {
   return prettier.format(
     `
-    interface SVGRProps {
-      title?: string;
-      titleId?: string;
-    }
     declare const ${svgName}: ({
       title,
       titleId,
       ...props
-    }: React.SVGProps<SVGSVGElement> & SVGRProps) => JSX.Element;
+    }: {
+      title?: string;
+      titleId?: string;
+    } & React.SVGProps<SVGSVGElement>) => JSX.Element;
```

Comment:
TypeScript has trouble creating downstream definitions when `SVGRProps` is used due to it being an interface internal to this file, so having the structure inline remedies this issue